globals/matchmaking_utils.go

```diff
@@ -231,11 +388,14 @@ func FindSessionByMatchmakeSession(pid *types.PID, searchMatchmakeSession *match
 
 // FindSessionsByMatchmakeSessionSearchCriterias finds a gathering that matches with the given search criteria
 func FindSessionsByMatchmakeSessionSearchCriterias(pid *types.PID, searchCriterias []*match_making_types.MatchmakeSessionSearchCriteria, gameSpecificChecks func(searchCriteria *match_making_types.MatchmakeSessionSearchCriteria, matchmakeSession *match_making_types.MatchmakeSession) bool) []*CommonMatchmakeSession {
-	candidateSessions := make([]*CommonMatchmakeSession, 0, len(Sessions))
+	sessionsMutex.RLock()
+	defer sessionsMutex.RUnlock()
+	
+	candidatesessions := make([]*CommonMatchmakeSession, 0, len(sessions))
```

Comment:
Why was this casing changed? The variable name is made of 2 words, "candidate" and "sessions"

Reply:
I think the casing was not changed, it was like that already.